sdk/containerregistry/azure-containerregistry/azure/containerregistry/_container_registry_client.py

```diff
@@ -224,12 +224,12 @@ def list_manifest_properties(self, repository, **kwargs):
 
         :param str repository: Name of the repository
         :keyword order_by: Query parameter for ordering by time ascending or descending
-        :paramtype order_by: :class:`~azure.containerregistry.ManifestOrder` or str
+        :paramtype order_by: ~azure.containerregistry.ArtifactManifestOrder or str
         :keyword results_per_page: Number of repositories to return per page
         :paramtype results_per_page: int
-        :returns: An iterable of :class:`~azure.containerregistry.ArtifactManifestProperties`
+        :returns: An iterable of ~azure.containerregistry.ArtifactManifestProperties
```

Comment:
This one actually _should_ keep `:class:`, "returns" is a sentence tag, whereas "rtype" is the type tag.